.github/workflows/_docker-build.yml

```diff
@@ -57,7 +57,9 @@ jobs:
           import hashlib
           import os
 
-          hash = hashlib.sha256('''${{ inputs.build-args }}'''.encode())
+          hash = hashlib.sha256('''${{ inputs.namespace-repository }}
```

Comment:
I had to include these otherwise the generated hash is the same when building the SuperLink and SuperNode (because they have the same build-args) The id is used for the upload and download artifact action.